### PR TITLE
[fix](regression) add 'if not exists' to 'create table' to support parallel test (#13576)

### LINKED ISSUE #13576: [Bug] regression test failed for duplicated table creation
### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/incubator-doris/issues?q=is%3Aissue) and found no similar issues.

### Version

all

### What's Wrong?

regression test failed for duplicated table creation after being made parallel execution.

### What You Expected?

correct

### How to Reproduce?

_No response_

### Anything Else?

_No response_

### Are you willing to submit PR?

- [X] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by freemandealer**
Signed-off-by: freemandealer <freeman.zhang1992@gmail.com>

# Proposed changes

Issue Number: close #13576

## Problem summary

Describe your changes.

## Checklist(Required)

1. Does it affect the original behavior: 
    - [ ] Yes
    - [X] No
    - [ ] I don't know
2. Has unit tests been added:
    - [ ] Yes
    - [ ] No
    - [X] No Need
3. Has document been added or modified:
    - [ ] Yes
    - [ ] No
    - [X] No Need
4. Does it need to update dependencies:
    - [ ] Yes
    - [X] No
5. Are there any changes that cannot be rolled back:
    - [ ] Yes (If Yes, please explain WHY)
    - [X] No

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by hello-stephen** [comment]
TeamCity pipeline, clickbench performance test result:
 the sum of best hot time: 38.63 seconds
 load time: 602 seconds
 storage size: 17155053087 Bytes
 https://doris-community-test-1308700295.cos.ap-hongkong.myqcloud.com/tmp/20221022131244_clickbench_pr_32766.html

**Review by morningman — DISMISSED** [review]
LGTM

**Comment by morningman** [comment]
Something wrong with the replacement like:
```
Exception:
java.sql.SQLException: errCode = 2, detailMessage = Syntax error in line 1:
SHOW CREATE TABLE IF NOT EXISTS `test_ctas1`
                  ^
Encountered: IF
Expected: IF is keyword, maybe `IF`
```

File: regression-test/data/query/show/test_array_show_create.out [hunk 1]
```diff
@@ -1,4 +1,4 @@
...
+test_array_show_create	CREATE TABLE IF NOT EXISTS `test_array_show_create` (\n  `k1` int(11) NULL,\n  `k2` array<smallint(6)> NOT NULL,\n  `k3` array<int(11)> NOT NULL,\n  `k4` array<bigint(20)> NOT NULL,\n  `k5` array<char(1)> NOT NULL,\n  `k6` array<varchar(20)> NULL,\n  `k7` array<date> NOT NULL,\n  `k8` array<datetime> NOT NULL,\n  `k9` array<float> NOT NULL,\n  `k10` array<double> NOT NULL,\n  `k11` array<decimal(20, 6)> NULL\n) ENGINE=OLAP\nDUPLICATE KEY(`k1`)\nCOMMENT 'OLAP'\nDISTRIBUTED BY HASH(`k1`) BUCKETS 1\nPROPERTIES (\n"replication_allocation" = "tag.location.default: 1",\n"in_memory" = "false",\n"storage_format" = "V2",\n"disable_auto_compaction" = "false"\n);
```
Comment (pingchunzhang) [inline regression-test/data/query/show/test_array_show_create.out:3]: even create table with “IF NOT EXISTS”，which will not be displayed when show create table

File: regression-test/data/ddl_p0/test_ctl.out [hunk 2]
```diff
@@ -1,4 +1,4 @@
 -- This file is automatically generated. You should know what you did if you want to edit this
 -- !select --
-test_ctl1	CREATE TABLE `test_ctl1` (\n  `test_varchar` varchar(150) NULL,\n  `test_datetime` datetime NULL,\n  `test_default_timestamp` datetime NULL DEFAULT CURRENT_TIMESTAMP\n) ENGINE=OLAP\nUNIQUE KEY(`test_varchar`)\nCOMMENT 'OLAP'\nDISTRIBUTED BY HASH(`test_varchar`) BUCKETS 3\nPROPERTIES (\n"replication_allocation" = "tag.location.default: 1",\n"in_memory" = "false",\n"storage_format" = "V2",\n"disable_auto_compaction" = "false"\n);
+test_ctl1	CREATE TABLE IF NOT EXISTS `test_ctl1` (\n  `test_varchar` varchar(150) NULL,\n  `test_datetime` datetime NULL,\n  `test_default_timestamp` datetime NULL DEFAULT CURRENT_TIMESTAMP\n) ENGINE=OLAP\nUNIQUE KEY(`test_varchar`)\nCOMMENT 'OLAP'\nDISTRIBUTED BY HASH(`test_varchar`) BUCKETS 3\nPROPERTIES (\n"replication_allocation" = "tag.location.default: 1",\n"in_memory" = "false",\n"storage_format" = "V2",\n"disable_auto_compaction" = "false"\n);
```
Comment (pingchunzhang) [inline regression-test/data/ddl_p0/test_ctl.out:3]: even create table with “IF NOT EXISTS”，which will not be displayed when show create table， u should move “IF NOT EXISTS”

File: regression-test/data/ddl_p0/test_ctas.out [hunk 3]
```diff
@@ -1,6 +1,6 @@
 -- This file is automatically generated. You should know what you did if you want to edit this
 -- !select --
-test_ctas1	CREATE TABLE `test_ctas1` (\n  `test_varchar` varchar(150) NULL,\n  `test_datetime` datetime NULL,\n  `test_default_timestamp` datetime NULL DEFAULT CURRENT_TIMESTAMP\n) ENGINE=OLAP\nDUPLICATE KEY(`test_varchar`)\nCOMMENT 'OLAP'\nDISTRIBUTED BY HASH(`test_varchar`) BUCKETS 10\nPROPERTIES (\n"replication_allocation" = "tag.location.default: 1",\n"in_memory" = "false",\n"storage_format" = "V2",\n"disable_auto_compaction" = "false"\n);
+test_ctas1	CREATE TABLE IF NOT EXISTS `test_ctas1` (\n  `test_varchar` varchar(150) NULL,\n  `test_datetime` datetime NULL,\n  `test_default_timestamp` datetime NULL DEFAULT CURRENT_TIMESTAMP\n) ENGINE=OLAP\nDUPLICATE KEY(`test_varchar`)\nCOMMENT 'OLAP'\nDISTRIBUTED BY HASH(`test_varchar`) BUCKETS 10\nPROPERTIES (\n"replication_allocation" = "tag.location.default: 1",\n"in_memory" = "false",\n"storage_format" = "V2",\n"disable_auto_compaction" = "false"\n);
```
Comment (pingchunzhang) [inline regression-test/data/ddl_p0/test_ctas.out:3]: even create table with “IF NOT EXISTS”，which will not be displayed when show create table， u should move “IF NOT EXISTS”

File: regression-test/data/query/show/test_array_show_create.out (hunk 1 above)
Comment (pingchunzhang) [inline regression-test/data/query/show/test_array_show_create.out:3]: even create table with “IF NOT EXISTS”，which will not be displayed when show create table， u should move “IF NOT EXISTS”

**Review by pingchunzhang — COMMENTED** [review]
.out file no need add key work "if not exists"

**Review by dataroaring — APPROVED** [review]
LGTM

**Pull request merged**